ios/build.sh

```diff
@@ -66,6 +67,7 @@ while [[ $# -gt 0 ]] ; do
         -no-codesign)
             CODE_SIGN_IDENTITY="CODE_SIGN_IDENTITY="
```

Comment:
`CODE_SIGN_IDENTITY` and `CODE_SIGNING_REQUIRED` are unused

Reply:
Oh right, I can erase those now.  Forgot to do that when I got the unified variable working.